packages/fluentui/react-northstar/src/components/Tooltip/Tooltip.tsx

```diff
@@ -228,7 +230,12 @@ const Tooltip: React.FC<TooltipProps> &
   const element = (
     <>
       {triggerElement && (
-        <Ref innerRef={triggerRef}>{React.cloneElement(triggerElement, getA11Props('trigger', triggerProps))}</Ref>
+        <Ref innerRef={triggerRef}>
+          {React.cloneElement(triggerElement, {
+            className: Tooltip.slotClassNames.trigger,
```

Comment:
Let's avoid doing this as it will silently override existing `className` on a `trigger`

Reply:
So what do you suggest? Doing a `cx` composition of classes for that? I need to get that triggerElement in tests so I need the slot class.

Reply:
`cx` can work, but in this case it should be done also in `Popup`. However, I don't like this option as `trigger` is not a real slot.

If it's only for tests I don't see any blocker to use `Button.className` or use a specific classname on it.

Reply:
removed, using button class now.